test/images/scripts/rhel-7.setup

```diff
@@ -84,6 +84,15 @@ sos \
 tuned \
 "
 
+# If the RHEL version is new enough, install setroubleshoot-server
```

Comment:
> Wouldn't we just change this once our main image is new enough? It's not like we'll need to go back and create older images again.

The problem is that for quite a while, we'll be using old and new images in parallel - i.e. QE will use early 7.3 composes and want to test those, while upstream cockpit still uses 7.2.